src/mame/layout/j_lal.lay

```diff
@@ -0,0 +1,409 @@
+<?xml version="1.0"?>
+<!--
+Lite a Line
+license:CC0
+-->
+<mamelayout version="2">
+	<element name="matrixlamp">
+		<rect state ="0">
+			<bounds x="0" y="0" width="7" height="7" />
+			<color red="0.7" green="0.7" blue="0.7" />
+		</rect>
+		<rect state ="1">
+			<bounds x="0" y="0" width="7" height="7" />
+			<color red="0.95" green="0.95" blue="0.43" />
+		</rect>
```

Comment:
I’m not sure what you’re trying to do with the bounds here – the internal coordinate system of the element will adjust to the bounds of all components.  If you’ve only got `rect` components that fill the element, you may as well omit the bounds and take the default unit square.

Reply:
Removed these. They came from whichever layout I copy-pasted all of mine from, wasn't paying much attention to these things.